dff/config/superset_dashboard/dashboards/DFF_statistics_dashboard_1.yaml

```
@@ -0,0 +1,1315 @@
+      chartId: 13
+      height: 60
+      sliceName: Flow visit ratio monitor
+      uuid: ba02528b-184b-4304-b027-f2b7d9011ab0
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-6zE8noCIsx
+    - ROW-ZTVWOu2o0
+    type: CHART
+  CHART-explore-22-1:
+    children: []
+    id: CHART-explore-22-1
+    meta:
+      chartId: 15
+      height: 61
+      sliceName: Terminal labels
+      sliceNameOverride: Terminal labels monitor
+      uuid: cf066e41-a9e8-4f54-a875-ebf4da350b59
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-6zE8noCIsx
+    - ROW-LhKWfqM7V
+    type: CHART
+  CHART-explore-9-1:
+    children: []
+    id: CHART-explore-9-1
+    meta:
+      chartId: 6
+      height: 105
+      sliceName: Node visits [ratio]
+      sliceNameOverride: Node visit distribution [ratio]
+      uuid: f9fb7893-3533-4519-bbc4-1f4853f380e1
+      width: 6
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-Gw0Ffh1lG
+    - ROW-uSb7snkjI
+    type: CHART
+  CHART-mYC2udeF3a:
+    children: []
+    id: CHART-mYC2udeF3a
+    meta:
+      chartId: 9
+      height: 50
+      sliceName: Service load [users]
+      uuid: b5d43314-514c-464e-9fcc-f897e3ae0963
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-6zE8noCIsx
+    - ROW-Ae7v1prsp2
+    type: CHART
+  CHART-wci7CHiza6:
+    children: []
+    id: CHART-wci7CHiza6
+    meta:
+      chartId: 12
+      height: 66
+      sliceName: Transition counts
+      sliceNameOverride: General transitions distribution
+      uuid: 9fcc7cc1-9257-4c0d-b377-b3a60a8bf3df
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-Gw0Ffh1lG
+    - ROW-UiCWq3ix1
+    type: CHART
+  DASHBOARD_VERSION_KEY: v2
+  GRID_ID:
+    children:
+    - TABS-Xoi5oUBxZI
+    id: GRID_ID
+    parents:
+    - ROOT_ID
+    type: GRID
+  HEADER_ID:
+    id: HEADER_ID
+    meta:
+      text: DFF statistics dashboard
+    type: HEADER
+  MARKDOWN-8Q9BhcEwva:
+    children: []
+    id: MARKDOWN-8Q9BhcEwva
+    meta:
+      code: '## Current topic
+
+
+        Assuming that we extract the current topic as a slot on each dialog turn,
+        we can build a bar chart to see how various dialog topics are distributed.
+        The first of the two charts below aggregates the counts over dialog turns,
+        while the second one displays the distribution over time.'
+      height: 20
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-q4CdGUDlq
+    - ROW-K0fThyfqJq
+    type: MARKDOWN
+  MARKDOWN-HncT4Y_WmV:
+    children: []
+    id: MARKDOWN-HncT4Y_WmV
+    meta:
+      code: '## Flow visit ratio monitor
+
+
+        Thus chart aggregates the ratio of visits for each flow over the whole period
+        of service uptime. The aggregation period can be set to shorter time spans
+        using the filters on the left.'
```

Not necessarily to shorter ones:
```suggestion
        of service uptime. The aggregation period can be set using the filters on the left.'
```